### PR TITLE
Add PHP Storm phpcs instructions

### PR DESCRIPTION
**Pull request by verytwisty**
writing up slack conversation with Justin

## What does this do/fix?

Adds documentation for setting up PHPCS with PHP Strom

## Tests

Does this have tests?

- [ ] Yes
- [x] No, this doesn't need tests because it's documentation
- [ ] No, I need help figuring out how to write the tests.

### CODE REVIEW

File: docs/tooling/phpcs.md [hunk 1]
```diff
@@ -19,4 +19,28 @@ This setting will display the exact ruleset being used to highlight an error in
...
+
+6. Click the PHP_CodeSniffer inspection link
+
+7. In the new dialogue box that opens make sure the PHP_Codesniffer validation is checked. 
+
+8. In options section scroll to the coding standard dropdown and pick `custom`
+
+9. click the `...` button to open a dialogue box to select the `phpcs.xml.dist` in the root of the square one directory
+
+## Make sure project won't commit with errors
```
Comment (defunctl) [inline docs/tooling/phpcs.md:40]: Can instead link all of this to https://github.com/moderntribe/square-one/blob/main/docs/tooling/git-hooks.md#automatic-git-hooks so we don't have to maintain the same document in two places.

File: docs/tooling/phpcs.md [hunk 2]
```diff
@@ -19,4 +19,28 @@ This setting will display the exact ruleset being used to highlight an error in
 
 ## PhpStorm
 
-TODO
+1. In the Settings/Preferences, navigate to PHP | Quality Tools.
+
+2. On the Quality Tools page that opens, expand the PHP_CodeSniffer area and click the Browse button next to the Configuration list.
+
+3. In the PHP_CodeSniffer dialog that opens, go to: `project-folder/bin/`
```
Comment (defunctl) [inline docs/tooling/phpcs.md:26]:
```suggestion
3. In the PHP_CodeSniffer dialog that opens, go to: `project-folder/vendor/bin/`
```

File: docs/tooling/phpcs.md [hunk 3]
```diff
@@ -19,4 +19,28 @@ This setting will display the exact ruleset being used to highlight an error in
...
+
+5. Open the PHP Code Beutifier and Fixer settings and in the same folder select `phpcbf` shortcut file.
+
+6. Click the PHP_CodeSniffer inspection link
+
+7. In the new dialogue box that opens make sure the PHP_Codesniffer validation is checked. 
+
+8. In options section scroll to the coding standard dropdown and pick `custom`
+
+9. click the `...` button to open a dialogue box to select the `phpcs.xml.dist` in the root of the square one directory
```
Comment (defunctl) [inline docs/tooling/phpcs.md:38]: Would add another step here to only tell it to check `php` extensions, by removing the `js,css` ones.

**Review by defunctl — CHANGES_REQUESTED** [review]
Couple of changes, would like screenshots at some point but this is still way better than a TODO :)

**Comment by verytwisty** [comment]
@defunctl made those changes and added some images :-)

**Review by nickpelton — APPROVED** [review]
Docs! ❤️  🚀

**Review by defunctl — APPROVED** [review]
Looks great! :rocket:

**Pull request merged**